modules/structs/user.go

```diff
@@ -17,6 +17,8 @@ type User struct {
 	ID int64 `json:"id"`
 	// the user's username
 	UserName string `json:"login"`
+	// the user's login name
```

Comment:
I think this can be easy misunderstood, it should mention that it's the username from the login source and is by default empty

Reply:
Resolved